### DISCUSSION TITLE
Minor issue in the Spanish translation

### DISCUSSION THREAD

**Discussion by Panquesito7** [WeblateOrg/weblate#5689] [General] [answered]
In the Spanish translation, there's the `Libre` word, but the translation is the same as in the English translation, therefore, it reports an [error](https://hosted.weblate.org/translate/weblate/application/es/?q=has:dismissed-check). I suggest we change the English string translation to `Free` and keep the Spanish string translation as-is.

What do you guys think? 🙂

**Comment by nijel** [comment, marked as answer]
The check is dismissed, what is okay - the checks can produce false positives, and it is expected that translators will dismiss such checks.

**Reply by Panquesito7** [reply]
Thanks! However, some other strings in Spanish are the same in plural and singular. Is there a way to ignore that? Thanks though. 🙂

**Reply by nijel** [reply]
You can dismiss any check (unless it is enforced in the component configuration).
